src/common/constants.tsx

```diff
@@ -1757,3 +1757,78 @@ export const HEADER_CONTENT_TYPES = {
 } as const;
 
 export const ADMIN_USER_TYPES = ["DistrictAdmin", "StateAdmin"] as const;
+
+export const ENCOUNTER_DIET_PREFERENCE = [
+  "vegetarian",
+  "diary_free",
+  "nut_free",
+  "gluten_free",
+  "vegan",
+  "halal",
+  "kosher",
+  "none",
+] as const;
+
+export const ENCOUNTER_STATUS = [
+  "planned",
+  "in_progress",
+  "on_hold",
+  "discharged",
+  "completed",
+  "cancelled",
+  "discontinued",
+  "entered_in_error",
+  "unknown",
+] as const;
+
+export const ENCOUNTER_CLASS = [
+  "imp",
+  "amb",
+  "obsenc",
+  "emer",
+  "vr",
+  "hh",
+] as const;
+
+export const ENCOUNTER_PRIORITY = [
+  "ASAP",
+  "callback_results",
+  "callback_for_scheduling",
+  "elective",
+  "emergency",
+  "preop",
+  "as_needed",
+  "routine",
+  "rush_reporting",
+  "stat",
+  "timing_critical",
+  "use_as_directed",
+  "urgent",
+] as const;
+
+export const ENCOUNTER_ADMIT_SOURCE = [
+  "hosp_trans",
+  "emd",
+  "outp",
+  "born",
+  "gp",
+  "mp",
+  "nursing",
+  "psych",
+  "rehab",
+  "other",
+] as const;
+
+export const ENCOUNTER_DISCHARGE_DISPOSITION = [
+  "home",
+  "alt_home",
+  "other_hcf",
+  "hosp",
+  "long",
+  "aadvice",
+  "exp",
+  "psy",
+  "rehab",
+  "snf",
+  "oth",
+] as const;
```

Comment:
we already have most of these defined in https://github.com/ohcnetwork/care_fe/blob/develop/src/types/emr/encounter.ts#L6-L67

Let's convert that to array and update the type to be

```
export const ENCOUNTER_DISCHARGE_DISPOSITION = [...] as const;

export type EncounterDischargeDisposition = (typeof ENCOUNTER_DISCHARGE_DISPOSITION)[number];
```

Reply:
let's move this to the types folder instead of keeping it in constants.tsx.

That's where we store the new things.